packages/nextjs/services/web3/wagmi-burner/burnerWalletConfig.ts

```diff
@@ -30,10 +34,6 @@ export const burnerWalletConfig = ({ chains }: BurnerWalletOptions): Wallet => (
     return false;
   },
   createConnector: () => {
```

Comment:
createConnector: createBurnerConnector ?

Reply:
Yup also fixed this at [81c995b](https://github.com/scaffold-eth/scaffold-eth-2/pull/700/commits/81c995bee75680e05c8257ba7af89f225127cf5d)